src/block/tile/MonsterSpawner.php

```diff
@@ -130,4 +146,125 @@ protected function addAdditionalSpawnData(CompoundTag $nbt) : void{
 
 		$nbt->setFloat(self::TAG_ENTITY_SCALE, $this->displayEntityScale);
 	}
+
+	public function onUpdate() : bool{
```

Comment:
I dont think this method need to be here. We want to keep tile only for storing data. His place are in the block class.

Reply:
If i remember, the ambition was to moove the furnace logics etc into the block. Maybe i'm wrong

Reply:
So where should I put the logic? It will still have to be called from onUpdate() if its in the block class

Reply:
We should put the logic into the block. The tile need only to be used for storing or actions which arent possible with blocks. As we can see, in the furnace class, there is a TODO, to moove all the logic into the block.
That makes sense because this `onUpdate` function is only called in the block class, so why putting it into the tile ?